app/test/test_audio_similarity.py

```diff
@@ -291,34 +291,34 @@ def test_audio_model_confirmed_no_match(self):
         self.assertEqual(second_case['context'], [{'blah': 3, 'has_custom_id': True, 'project_media_id': 1457}])
         self.assertLessEqual(second_case['score'], 0.1)
 
-    def test_search_by_context(self):
-        url = 'file:///app/app/test/data/test_audio_1.mp3'
-        hash_key = "blah"
-        audio = Audio(chromaprint_fingerprint=first_print, doc_id="blah32", url="http://blah.com", context=[{'blah': 16, 'has_custom_id': True, 'project_media_id': 12343}])
-        db.session.add(audio)
-        db.session.commit()
-        results = self.model.search_by_context({"blah": 16})
-        self.assertEqual(results[0]['doc_id'], 'blah32')
-        self.assertEqual(results[0]['url'], "http://blah.com")
-        self.assertEqual(results[0]['context'], [{'blah': 16, 'has_custom_id': True, 'project_media_id': 12343}])
-        results = self.model.search_by_context({"blah": 2})
-        self.assertEqual(results, [])
-
-    def test_handle_save_error(self):
-        with patch('app.main.lib.shared_models.audio_model.AudioModel.save', ) as mock:
-            mock.return_value = False
-            url = 'file:///app/app/test/data/test_audio_1.mp3'
-            result = self.model.add({"url": url, 'doc_id': "Y2hlY2stcHJvamVjdF9tZWRpYS01NTQ1NzEtdmlkZW8", "context": {"has_custom_id": True}})
-            self.assertIsInstance(result, dict)
-            self.assertEqual(False, result['success'])
-
-    def test_handle_http_error(self):
-        with patch('app.main.lib.shared_models.audio_model.AudioModel.save', ) as mock:
-            url = 'file:///app/app/test/data/test_audio_1.mp3'
-            mock.side_effect = urllib.error.HTTPError(url, 420, "HTTP ERROR HAPPENED", None, None)
-            result = self.model.add({"url": url, 'doc_id': "Y2hlY2stcHJvamVjdF9tZWRpYS01NTQ1NzEtdmlkZW8", "context": {"has_custom_id": True}})
-            self.assertIsInstance(result, dict)
-            self.assertEqual(False, result['success'])
-
+    # def test_search_by_context(self):
```

Comment:
are these tests intended to be commented?

Reply:
Yep - we no longer serve the audio model directly, so tests that test for that are moot